content/influxdb/clustered/process-data/downsample/client-libraries.md

```diff
@@ -42,57 +43,59 @@ Use `pip` to install the following dependencies:
 pip install influxdb3-python pandas
 ```
 
-## Prepare InfluxDB databases
+## Prepare InfluxDB buckets
 
-The downsampling process involves two InfluxDB databases.
-Each database has a [retention period](/influxdb/clustered/reference/glossary/#retention-period)
+The downsampling process involves two InfluxDB buckets.
+Each bucket has a [retention period](/influxdb/cloud-serverless/reference/glossary/#retention-period)
 that specifies how long data persists in the database before it expires and is deleted.
```

Comment:
Revert the remaining changes on this doc. InfluxDB Clustered:

- `bucket` -> `database`
- `influxdb/cloud-serverless` -> `influxdb/clustered`.